src/sass/app_2/modules/trading.scss

```diff
@@ -258,6 +258,48 @@
                 }
             }
         }
+        &.barriers {
+            .fieldset-header {
+                text-align: center;
+                height: 1.2rem;
+                margin-top: 2px;
+
+                .field-info {
+                    width: 100%;
+                    left: 0 !important;
+                }
+            }
+            .input-field {
+                input {
+                    padding-left: 3px;
+                }
+                &.multiple {
+                    display: inline-block;
+                    width: 46%;
```

Comment:
Maybe we would be better off using something like `flexbox` or `cssgrids` since we just want a two column design with even widths.